repo/init.go

```diff
@@ -168,6 +168,45 @@ func addConfigExtensions(repoRoot string, testnet bool) error {
 			LowFeeDefault:    20,
 			TrustedPeer:      "",
 		}
+		ws = schema.WalletsConfig{
+			BTC: schema.CoinConfig{
+				Type:             "API",
+				API:              "https://btc.bloqapi.net/insight-api",
+				APITestnet:       "https://test-insight.bitpay.com/api",
```

Comment:
`API`, `APITestnet`, `FeeAPI` are all specific to the wallet in use and is not specific to the coin. Imagine where one API wallet might use one API and another wallet uses another API. I'd prefer that `[]WalletsConfig` contains `WalletConfig` and `[]CoinConfigs` contain `CoinConfig`. This way each wallet can be configured separately as needed.

Reply:
What would go into `CoinConfig` in your mind? The data here is really specific to the implementation as some other implementation could require different settings.

Reply:
Things which are universal to the coin: `economyFee`, `*Fee`, the Divisibility for crypto listings if we need to coin-specific values, perhaps?

Reply:
Things like divisibility which are unique to the coin should probably be part of the coin interface and hardcoded rather than a configurable option. 

I think the config is only for user settings how they want their wallet to run. 

On those lines, fees are not really coin-specific per se, users might want to modify the defaults (and I know a couple users who have).